service_container/alias_private.rst

```diff
@@ -57,12 +57,11 @@ What makes private services special is that, if they are only injected once,
 they are converted from services to inlined instantiations (e.g. ``new PrivateThing()``).
 This increases the container's performance.
 
-Now that the service is private, you *should not* fetch the service directly
+Now that the service is private, you *must not* fetch the service directly
```

Comment:
Ideally I'd like to say: *"you can't fetch..."* but since you can or cannot depending on some other circumstances, I guess your reword is the best we can do for now.